packages/blade/src/components/Input/BaseInput/BaseInput.tsx

```diff
@@ -677,6 +704,33 @@ const getDescribedByElementId = ({
   return '';
 };
 
+const FocusRingWrapper = styled(BaseBox)<{ currentInteraction: ActionStates }>(
```

Comment:
what did we do about that layout shift because of border we were discussing?

Reply:
Used box shadow for border and added a wrapper that uses our focus ring styles (outline-based)